libs/blog/partners/ui-partners/src/lib/partners/partners.component.ts

```diff
@@ -1,13 +1,20 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
 import { TranslocoDirective } from '@ngneat/transloco';
 
+import { Partner } from '@angular-love/blog/shared/types';
+
 import { PartnersListComponent } from '../partners-list/partners-list.component';
 
+type title = 'homePage.partners' | 'homePage.mainPartner';
+
 @Component({
   selector: 'al-partners',
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
   imports: [PartnersListComponent, TranslocoDirective],
   templateUrl: './partners.component.html',
 })
-export class PartnersComponent {}
+export class PartnersComponent {
+  title = input.required<title>();
```

Comment:
you can use just 'string' as a type. I don't see any advantages for strictly typing that

Reply:
My purpose was to narrow it down. there is no extra advantage